etc/deploy/aws.sh

```diff
@@ -239,10 +239,11 @@ get_k8s_master_domain() {
 check_kops_version() {
   which kops
   KOPS_VERSION="$( kops version | awk '{print $2}' )"
-  echo "${KOPS_VERSION#1.} >= 7.1" | bc
-  if [[ "$( echo "${KOPS_VERSION#1.} >= 7.1" | bc )" -ne 1 ]]; then
+  KOPS_VERSION="${KOPS_VERSION#1.}" # 1.7.0 -> 7.0 (so that it's a binary number that bc can read)
```

Comment:
Is this line redundant? It seems like this operation is done on line 244 either way (after the alpha/beta suffix is removed).